<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Evolver</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="buttonWrapperWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="text">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="originalImageText">
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="text">
         <string>Original Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentImageText">
        <property name="text">
         <string>Current Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="originalImageLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>3</width>
          <height>1</height>
         </size>
        </property>
        <property name="text">
         <string>Load Something Yo</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="currentImageLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Start Something</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttons">
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Load Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadSVGButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load SVG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="setShapesButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Set Used Shapes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backgroundColorButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Set Background Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aboutButton">
        <property name="text">
         <string>About</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="saveStepsBox">
        <property name="text">
         <string>Save Incrementally</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="updateImageBox">
        <property name="text">
         <string>Update Images</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadSVGButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadSVG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveStepsBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>saveIncrementalChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>updateStatusBar(QString)</signal>
   <receiver>statusBar</receiver>
   <slot>showMessage(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>toggleMutation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setShapesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setShapes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backgroundColorButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeBackground()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateImageBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateImageToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>754</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>updateStatusBar(QString)</signal>
  <slot>loadImage()</slot>
  <slot>loadSVG()</slot>
  <slot>saveIncrementalChanged(bool)</slot>
  <slot>saveImage()</slot>
  <slot>toggleMutation()</slot>
  <slot>changeBackground()</slot>
  <slot>setShapes()</slot>
  <slot>showAbout()</slot>
  <slot>updateImageToggled(bool)</slot>
 </slots>
</ui>
